ci/docker-entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+set -e
+
+# if the first argument look like a parameter (i.e. start with '-'), run Envoy
+if [ "${1#-}" != "$1" ]; then
+	set -- envoy "$@"
+fi
+
+if [ "$1" = 'envoy' ]; then
```

Comment:
Why condition here? Seems like we should always be running something that understands Envoy CLI args here.

Reply:
So the idea with this is that sort of "by default" you get Envoy, but if you want to run something else, say you just want a shell to poke around, you can do that too, like:
```
docker run --rm -it envoyproxy/envoy bash
```